contest/templates/contest/contestInfo.html

```diff
@@ -52,16 +53,20 @@ <h3 class="modal-title">{{ contest.cname }}</h3>
             {% endfor %}
           </tbody>
         </table>
+        {% if user|has_auth:contest.id and contest.problem.count %}
+        <form method="POST" action="{% url 'contest:rejudge' %}">
+          {% csrf_token %}
+          <input type="hidden" name="contest" value="{{ contest.id}}">
+          <button type="submit button" class="btn btn-warning btn-block">
```

Comment:
Could we set up alert dialog when rejudge all submission of a problem or all problems?